metricbeat/module/apache/status/data.go

```diff
@@ -51,6 +50,29 @@ var (
 			"15": c.Float("Load15", s.Optional),
 		},
 	}
+
+	// Schema used till apache 2.4.12
+	schemaOld = s.Schema{
```

Comment:
We must make sure all these fields are also in the `fields.yml` documented.

Reply:
schemaOld contains a subset of the fields of the current schema.

Reply:
Well, I have removed the `server_uptime` field, it looks redundant to me with the `uptime` field. Also I think `uptime` shouldn't be a group, it should be enough with a `long` for this value, but changing from group to long looks fearful for backwards compatibility.

Reply:
Let's not introduce a breaking change here as this potentially goes out in a minor.

We should start a list of fields we want to change in the next master where we can break things.

For `server_uptime` vs `uptime` I'm not sure if one is from the actual host itself and the other one apache?

Reply:
Right, I revert these changes on `server_update`.